core/spec/models/spree/customer_return_spec.rb

```diff
@@ -83,6 +83,19 @@
     it { expect(Spree::CustomerReturn.whitelisted_ransackable_attributes).to eq(%w(number)) }
   end
 
+  describe '#display_pre_tax_total' do
+    subject { customer_return.display_pre_tax_total }
+
+    let(:pre_tax_amount)  { 15.0 }
+    let(:customer_return) { create(:customer_return, line_items_count: 2) }
+
+    before do
+      Spree::ReturnItem.where(customer_return_id: customer_return.id).update_all(pre_tax_amount: pre_tax_amount)
+    end
+
+    it { is_expected.to eq Spree::Money.new(pre_tax_amount * 2) }
```

Comment:
```suggestion
    it { is_expected.to eq '$ 30' }
```

we need to test the final output

Reply:
@tomurb also let's add a test for `currency` method as well